package.json

```diff
@@ -38,6 +38,7 @@
     "test:changed": "ts-node-script scripts/ci-test/test_changed.ts",
     "test:setup": "node tools/config.js",
     "test:saucelabs": "node scripts/run_saucelabs.js",
+    "test:cross-browser": "node scripts/run_cross_browser.js",
```

Comment:
If you use `ts-node` instead of `node`, you can make this script a Typescript file and get extra type checking on it. See `format` in line 54 for an example.